src/operation.js

```diff
@@ -154,8 +162,8 @@ export class Operation {
         if (!isUndefined(opts.limit) && !isString(opts.limit)) {
             throw new TypeError('limit argument must be of type String');
         }
-        let limit           = opts.limit ? opts.limit : "9223372036854775807";
-        attributes.limit    = Hyper.fromString(limit);
+        let limit           = opts.limit ? new BigNumber(opts.limit).mul(ONE) : new BigNumber(MAX_INT64);
```

Comment:
That's the only thing I'm not sure of. Are we using the same convention (https://github.com/stellar/stellar-protocol/issues/20) here too?